calc/src/mechanics/gen78.ts

```diff
@@ -716,9 +717,14 @@ export function calculateSMSS(
     move.category = attackSource.stats.atk > attackSource.stats.spa ? 'Physical' : 'Special';
   }
   const attackStat =
-    (move.named('Shell Side Arm') && defender.stats.def < defender.stats.spd) ? 'atk'
-    : move.named('Body Press') ? 'def'
-    : move.category === 'Special' ? 'spa' : 'atk';
+    move.named("Shell Side Arm") &&
```

Comment:
You should be able to just run `npm run fix` to handle these :)

Reply:
Thanks!  I wasn't sure how to format.